### PR TITLE
967 [SEO] Optimise styles.css

### LINKED ISSUE #967: [SEO] Optimise styles.css
In styles.css ( https://lifesciences.danaher.com/styles/styles.css ) can we move the blog or topics css to only that page template or change some of the css to js?

### PR DESCRIPTION
**Pull request by sharanagoudapatil**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #967 

Test URLs:
- Before: https://main--danaher-ls-aem--hlxsites.hlx.page/
- After: https://967-optimise-stylescss--danaher-ls-aem--hlxsites.hlx.page/us/en/blog/how-to-prepare-your-specimen-for-immunofluorescence-microscopy
- After: https://967-optimise-stylescss--danaher-ls-aem--hlxsites.hlx.page/us/en/news/scientists-can-study-neuronal-morphology-harnessing-the-power-of-ai
- After: https://967-optimise-stylescss--danaher-ls-aem--hlxsites.hlx.page/us/en/products/flow-cytometers/topics/sheath-fluid-flow-cytometry
- After: https://967-optimise-stylescss--danaher-ls-aem--hlxsites.hlx.page/us/en/products/mass-spectrometers/topics/what-is-mass-spectrometry
- After: https://967-optimise-stylescss--danaher-ls-aem--hlxsites.hlx.page/us/en/products/mass-spectrometers/topics/ion-analysis-in-mass-spectrometry

### CODE REVIEW

File: blocks/recent-articles/recent-articles.js [hunk 1]
```diff
@@ -27,12 +27,12 @@ export default async function decorate(block) {
...
   articles.forEach((article) => {
     const liEl = li(
-      { class: 'recent-articles-item py-2' },
+      { class: 'recent-articles-item pt-2 pb-0' },
       a(
         { class: 'block text-xs p-1.5 rounded transition-transform hover:bg-danaherpurple-50 hover:scale-[.99] hover:font-bold', href: makePublicUrl(article.path) },
         p({ class: 'text-sm font-medium pb-2 my-0' }, article.title),
         p(
-          { class: 'flex justify-between items-cente my-0' },
+          { class: 'flex justify-between items-cente mt-2 mb-0' },
```
Comment (amlan18041996) [inline blocks/recent-articles/recent-articles.js:35]: `items-cente` is not a proper class-name. Use `items-center`.

File: templates/topic/topic.js [hunk 2]
```diff
@@ -4,6 +4,26 @@ import { buildArticleSchema } from '../../scripts/schema.js';
...
+    section?.querySelector('h1')?.classList.add(...'my-2 py-2 px-0'.split(' '));
+    section?.querySelectorAll('h2').forEach((contentWrapper) => {
+      if (!['call-to-action', 'takeway'].includes(contentWrapper.parentElement.parentElement.parentElement.className)) {
+        contentWrapper.classList.add('pt-8');
+      }
+      if (!['container-two-col'].includes(contentWrapper.parentElement.className)) {
+        contentWrapper.classList.remove('pt-8');
+      }
+    });
+    section?.querySelectorAll('h3, h4').forEach((contentWrapper) => {
```
Comment (amlan18041996) [inline templates/topic/topic.js:17]: Use `section?.querySelectorAll('h3, h4')?.forEach((contentWrapper)...`

File: templates/topic/topic.js [hunk 3]
```diff
@@ -4,6 +4,26 @@ import { buildArticleSchema } from '../../scripts/schema.js';
...
+      contentWrapper.classList.add('pt-8');
+    });
+    section?.querySelectorAll('img').forEach((contentWrapper) => {
```
Comment (amlan18041996) [inline templates/topic/topic.js:20]: Use `section?.querySelectorAll('img')?.forEach((contentWrapper)...`

File: templates/topic/topic.js [hunk 4]
```diff
@@ -4,6 +4,26 @@ import { buildArticleSchema } from '../../scripts/schema.js';
...
+      contentWrapper.classList.add(...'w-full mt-10 mb-4'.split(' '));
+    });
+  });
+  main.querySelectorAll('p:not(.show-modal-btn)').forEach((contentWrapper) => {
```
Comment (amlan18041996) [inline templates/topic/topic.js:24]: Use `main.querySelectorAll('p:not(.show-modal-btn)')?.forEach((contentWrapper)...`

**Review by rgravitvl — APPROVED** [review]

**Comment by sharanagoudapatil** [comment]
Hi @davenichols-DHLS, 
Please find the Lighthouse scores screenshots for both main and current branch changes below,

https://967-optimise-stylescss--danaher-ls-aem--hlxsites.hlx.page/us/en/products/flow-cytometers/topics/sheath-fluid-flow-cytometry
image

https://main--danaher-ls-aem--hlxsites.hlx.page/us/en/products/flow-cytometers/topics/sheath-fluid-flow-cytometry
image

Please merge this PR if you are happy with these changes, otherwise will discuss on this for further improvements. Thanks.

**Review by amlan18041996 — APPROVED** [review]

**Review by davenichols-DHLS — APPROVED** [review]
FYI @mhaack .  Looks great to me.

**Pull request merged**